common/test/acceptance/tests/discussion/helpers.py

```diff
@@ -81,6 +81,7 @@ def add_user_to_cohort(self, course_fixture, username, cohort_id):
         """
         url = LMS_BASE_URL + "/courses/" + course_fixture._course_key + "/cohorts/{}/add".format(cohort_id)
         data = {"users": username}
+        course_fixture.headers['Content-type'] = 'application/x-www-form-urlencoded'
```

Comment:
[request.POST](https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/course_groups/views.py#L333) QueryDict is empty because `request.POST` in a django view will no longer include data posted via HTTP requests with non form-specific content-types in the header. In prior versions, data posted with content-types other than multipart/form-data or application/x-www-form-urlencoded would still end up represented in the request.POST attribute.